src/Pages/RandomBeers.js

```diff
@@ -0,0 +1,38 @@
+import { useEffect, useState } from "react"
+import Header from "../components/Header";
+import beersService from "../services/beersService";
+
+function RandomBeer() {
+    const [details, setDetails] = useState({});
+
+    useEffect(() => {
+        loadBeer()
+    })
```

Comment:
Don't forget to set a dependancy in your useEffect, otherwise it will reload endlessly: 
(that's also the reason why using useEffect ^^)

```
   useEffect(() => {
      loadBeer();
   }, []);
   
```